ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -958,6 +958,9 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
     }
 
+    /// The UIView of the `accessoryView`.
+    @objc open var trailingAccessoryView: UIView? { return accessoryTypeView }
```

Comment:
feels bit confusing. because we have like titletrailingview and subtitletrailingview. shouldn't it be like accessoryview? and if client has set custom accessory view, then it should return custom accessotryview instead of accessoryTypeView?

Reply:
Yea so I couldn't name accessoryView because that would mean I would have to override the property which I didn't have a problem with, but I cannot make it a get-only and would have to make it settable as well, which I don't think we want. And in terms of the custom accessory view, that can be displayed in conjunction with accessoryTypeView so that is a completely separate property

Reply:
Follow-up turn into function